ci/values-travis.yaml

```diff
@@ -23,6 +23,9 @@ persistence:
 influxdb:
   enabled: false
 
+rabbitmq:
+  enabled: true
```

Comment:
I think the default should be false?

Reply:
This is only for the testing, on the general values, rabbit is disabled. However, when installing tools from the toolshed and having multiple web/job handlers and you don't have a proper message queue (relay only on kombu + database), you see issues in all the handlers syncing new tool installs that require handlers restart. I have seen this across different Galaxy versions (19.05 to 20.01).

Reply:
I think that we should leave it enabled for testing.

Reply:
Sorry, I didn't notice that this was the travis values file. All good!

Reply:
Could you please revert then the request for changes in the review? Thanks. So that we can merge once it has been tested with the new 20.05 container image.

Reply:
I left it as is because the ERLANG cookie needs a fix though right?

Reply:
@pcm32 Is this good for another round of review, or is there more work to be done?

Reply:
sorry, I haven't had time to come back to this.